Sources/Layout/UIKit/UIView+Layout.swift

```diff
@@ -20,7 +20,8 @@ extension UIView {
     /// distinct states.
     ///
     /// - Note: Use ``layout(metrics:_:)`` to create a layout with one item and use ``layout(metrics:items:)`` for
-    ///   multiple items. Call ``Layout/addItems(_:)-59mxv`` to add more items to the layout.
+    ///   multiple items. Use ``Layout/addItem(_:)`` or ``Layout/addItems(items:)`` to add additional items to
```

Comment:
Curious—how we maintained linking behavior in the docs without the Xcode-provided ID appended to the method?

Reply:
Those methods were renamed, but the rename PR did not update these docs. With the rename there is no ambiguity, therefore the IDs are not necessary.